content/concepts/did-ddo.md

```diff
@@ -34,98 +43,120 @@ In Ocean, a DID is a string that looks like:
+- `status` -  stores status related fields [Status](#status)
+- `files` and `encryptedFiles` - stores file(s) informations [Files](#files)
+- `event` - stores the last event information [Event](#event)
 
-Asset metadata must be included as one of the objects inside the `"service"` array, with type `"metadata"`.
 
-## DDO Service Types
+## Metadata
+The object has the following attributes. 
 
-There are many possible service types for a DDO.
+| Attribute           | Type                  | Required | Description                                                                                                                                                                                       |
+| ------------------- | --------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **`description`**     | Text          | **Yes**       | Details of what the resource is. For a dataset, this attribute explains what the data represents and what it can be used for.|
+| **`copyrightHolder`** | Text          | No       | The party holding the legal copyright. Empty by default.   
+| **`name`**          | Text                  |**Yes**   | Descriptive name or title of the asset.                                                                                                                                                           |
+| **`type`**          | Text                  |**Yes**   | Asset type. Includes `"dataset"` (e.g. csv file), `"algorithm"` (e.g. Python script). Each type needs a different subset of metadata attributes. |
+| **`author`**        | Text                  |**Yes**   | Name of the entity generating this data (e.g. Tfl, Disney Corp, etc.).                                                                                                                            |
+| **`license`**       | Text                  |**Yes**   | Short name referencing the license of the asset (e.g. Public Domain, CC-0, CC-BY, No License Specified, etc. ). If it's not specified, the following value will be added: "No License Specified". |
+| **`links`**           | Array of Link | No       | Mapping of links for data samples, or links to find out more information. Links may be to either a URL or another Asset. We expect marketplaces to converge on agreements of typical formats for linked data: The Ocean Protocol itself does not mandate any specific formats as these requirements are likely to be domain-specific. The links array can be an empty array, but if there is a link object in it, then an "url" is required in that link object. |
+| **`inLanguage`**      | Text          | No       | The language of the content. Please use one of the language codes from the [IETF BCP 47 standard](https://tools.ietf.org/html/bcp47)|
```

Comment:
why not just `language` or `contentLanguage` , maybe it's just me but `inLanguage` doesn't sound right

Reply:
done